mappings/net/minecraft/datafixer/fix/EntityProjectileOwnerFix.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_4860 net/minecraft/datafixer/fix/EntityProjectileOwnerFix
+	METHOD <init> (Lcom/mojang/datafixers/schemas/Schema;)V
+		ARG 1 schema
```

Comment:
```suggestion
		ARG 1 parent
```

Reply:
Obviously this because it matches the params shown in DFU. Maybe we need someone to look over all the fixes and verify these have the right param names.

Reply:
Why though? The class name is very clearly `Schema`.

Reply:
So if you have a string parameter that serves as prefix, we'd rather name it `prefix` than `string`.

Reply:
I'll rephrase it, then: what about this parameter makes you think it should be called `parent`? And if all the use cases of this class refers to it as `parent`, why is the class itself not called `FixParent`?

To me a parent means the parameter is itself the parent node in a tree, and/or the same, or a super-type of `this`, the child element.

Reply:
Hmm, I think this should actually be
```suggestion
		ARG 1 outputSchema
```
forgot why i suggested `parent` before